packages/paste-core/components/button-group/stories/index.stories.tsx

```diff
@@ -66,6 +66,25 @@ export const PopoverButtons: Story = () => {
   );
 };
 
+export const ToggleButtons: Story = () => {
+  const [boldPressed, setBoldPressed] = React.useState(false);
+  const [italicPressed, setItalicPressed] = React.useState(false);
```

Comment:
suggestion(non-blocking): could be nice to have one of the buttons be pressed by default for VRT.

Reply:
suggestion: could we also add disabled and selected disabled states?